go/vt/mysqlctl/backupengine.go

```diff
@@ -112,7 +112,7 @@ type BackupRestoreEngine interface {
 var BackupRestoreEngineMap = make(map[string]BackupRestoreEngine)
 
 func init() {
-	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtcombo", "vttablet", "vttestserver", "vtctld", "vtctldclient", "vtexplain"} {
+	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtcombo", "vttablet", "vttestserver", "vtctld", "vtctldclient", "vtexplain", "vtbackup"} {
```

Comment:
I would think vtbackup, vttablet, and maybe vtctld need backup flags. We can pare this down later.

Reply:
Honestly, I have no idea. Maybe @deepthi or @rsajwani know 🤷‍♂️

Reply:
I am going ahead and merging this PR, because I want the latest lite image to be built so that I can continue testing https://github.com/planetscale/vitess-operator/pull/324. We can add these flags to vttablet and vtctld in a follow up PR if needed.

Reply:
We have to do the opposite, i.e, remove the other binaries: mysqlctl, vtexplain etc.
Not sure whether vtcombo and vttestserver are supposed to support backups.

Reply:
> vtbackup, vttablet, and m

Yes vttablet , vtctld will also need backup flags.